.github/workflows/deploy.yml

```diff
@@ -2,14 +2,20 @@ name: Deploy Styleguide
 
 on:
   push:
-    branches:
-      - master
+#    branches:
```

Comment:
Can we prune these comments?

Reply:
no, they are here for a reason. normally we want the branch protection, but for any change of this yml file, if i leave that in then it wont' run until i merge. SO I have to comment them out, get it working and finally, as a last step, I uncomment these before checking in,